plugins/services/src/js/structs/Pod.js

```diff
@@ -208,4 +208,28 @@ module.exports = class Pod extends Service {
   getVolumesData() {
     return new VolumeList({ items: this.get("volumeData") || [] });
   }
+
+  /**
+   * @override
+   */
+  getRegions() {
+    if (!this._regions) {
+      const regionCounts = this.getInstanceList()
+        .getItems()
+        .reduce((regions, instance) => {
+          const region = instance.getAgentRegion();
+          if (region !== "") {
+            regions[region] = regions[region] ? regions[region] + 1 : 1;
+          }
+
+          return regions;
+        }, {});
+
+      this._regions = Object.keys(regionCounts).sort(
+        (a, b) => regionCounts[b] - regionCounts[a]
```

Comment:
wonder why not natural sort? Couldn't find any explanation in the jira

Reply:
yeah, was my decision - I thought its a good idea to sort by tasks running on that instance 🤷‍♂️

Reply:
Couldn't it be confusing for a user as we don't show the actual count?

Reply:
The actual column is added here: https://github.com/dcos/dcos-ui/pull/3167
I'll change the implementation to use natural sort - have to rebase the feature branch anyway 🤷‍♂️